### ISSUE TITLE
error refreshing Plex -plex_debrid

### ISSUE THREAD

**Issue by spants** [I-am-PUID-0/pd_zurg#50]
When an item is found, Plex _Debrid should refresh Plex. However, an error is found in the logs:

ERROR	manager	Failed to execute hook on_library_update: error executing script: exit status 2; stderr: sh: can't open 'plex_update.sh': No such file or directory

Using Docker from Dockerhub running on unRaid

**Comment by mrtw85** [comment]
Great software but I am also receiving the same error, attached log

Jun  3, 2024 21:03:34 - ERROR - Zurg w/ RealDebrid subprocess: [0m	manager	Failed to execute hook on_library_update: error executing script: exit status 2; stderr: sh: can't open 'plex_update.sh': No such file or directory

[PDZURG-2024-06-03.log](https://github.com/user-attachments/files/15551503/PDZURG-2024-06-03.log)

**Comment by weagertron** [comment]
Also receiving this error and seems to be preventing the auto refreshing of the plex library. Not the end of the world but would be nice if it worked. I'm using the sample docker-compose file from the setup guide that includes plex.

I also tried rolling back as far as v2.0.0 of the docker image to see if that resolved the issue but sadly it didn't.

This is a great project and appreciate all the efforts involved, thank you!

**Comment by simlocker** [comment]
Having the same issue here.

I see there's a plex_update.sh here: https://github.com/debridmediamanager/zurg-testing/blob/main/plex_update.sh
Not sure if we could use that one, and also where to put it.

**Comment by I-am-PUID-0** [comment]
> Having the same issue here.
> 
> I see there's a plex_update.sh here: https://github.com/debridmediamanager/zurg-testing/blob/main/plex_update.sh Not sure if we could use that one, and also where to put it.

Yes. The zurg plex_update.sh can be used in lieu of `PLEX_REFRESH`. Disable `PLEX_REFRESH` and copy the plex_update.sh to the /RD directory.
